### DISCUSSION TITLE
New widgets

### DISCUSSION THREAD

**Discussion by tamaramegan** [Vizzuality/mangrove-atlas#341] [Ideas]
Hello @lhilarides @klongleywood !
We have prepared some ideas for the new widgets ahead of tomorrow's meeting.
You can take a look [at the presentation](https://docs.google.com/presentation/d/136vFGih-E4_C1ynyRRziiYcD1Am8u85C2SUkcrUADX4/edit?usp=sharing) and we will discuss them tomorrow.
Cheers!

**Comment by lyubovpencheva** [comment]
done
